src/profile_manager/tests/test_views.py

```diff
@@ -176,6 +179,23 @@ def test_profile_detail__student_marks_button(self):
         response = self.client.get(reverse('profiles:profile_detail', args=[s_pk]))
         self.assertContains(response, 'View your Mark Calculations')
 
+    def test_profile_own__correct_displayed_text(self):
```

Comment:
Now that this view is serving both profile_own and other profile_detail, there should also be a test to confirm profile_detail actually returns the correct profile (and not own).

Currently we are only checking for 200 response.